rclrs/src/node/mod.rs

```diff
@@ -96,7 +97,7 @@ impl Node {
         qos: QoSProfile,
     ) -> Result<Publisher<T>, RclReturnCode>
     where
-        T: MessageDefinition<T>,
```

Comment:
Why was the generic type parameter dropped?

Reply:
I just didn't see any reason why it should be there.